store/volume.go

```diff
@@ -4,9 +4,7 @@ package store
 // volumes stored in the store
```

Comment:
This extra level of abstraction doesn't give any gains. The functions exported here are only used by the `volume` package. I don't believe anyone else would want to use these helper functions. It's better to just not have these at all.